### ISSUE TITLE
Add ANSI styling to chat and disconnect messages when logged to terminal

### ISSUE THREAD

**Issue by stackotter** [stackotter/delta-client#110]
It's pretty self explanatory, currently chat and disconnect messages are just printed out plain (and disconnect messages don't even have the legacy formatting codes removed).

**Comment by ninjadev64** [comment]
Would we just need to modify ChatComponentColor to provide ANSI escape sequences for each enumeration, and then use `tokens.map(\.color.rawValue + \.string).joined()` in LegacyFormattedText#toString?

**Comment by stackotter** [comment]
Yeah essentially, although don’t make the rawValue ansi, just give the colour enum an ansiValue property (cause we might need more ‘raw’ values in future too)

**Comment by ninjadev64** [comment]
Alright, I'll do this one.
